data/manifest-override.yaml

```diff
@@ -9,3 +9,8 @@ Overwatch:
   # and its save locations are identical to Overwatch 2,
   # leading to duplicate backups.
   omit: true
+Counter-Strike Global Offensive:
```

Comment:
I just realized, this should be `'Counter-Strike: Global Offensive'` since [the article title](https://www.pcgamingwiki.com/wiki/Counter-Strike:_Global_Offensive) contains a colon.